WBooks/Library/LibraryViewController.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  LibraryViewController.swift
+//  WBooks
+//
+//  Created by Gaston Maspero on 13/06/2019.
+//  Copyright © 2019 Wolox. All rights reserved.
+//
+
+import UIKit
+import WolmoCore
+
+class LibraryViewController: UIViewController {
+    
+    private let margin: CGFloat = 20
+    
+    private var _booksController: BookTableController
```

Comment:
Why do you have the viewController and not the view? If you set BookTable view as subview of libraryView, it should be loaded when the parent is loaded